clang/test/CAS/fcas-include-tree-prefix-mapping.c

```diff
@@ -55,6 +57,27 @@
 
 // RUN: diff -u %t/cache-key %t/cache-key2
 
+// Check dependencies.
+
+// Baseline for comparison.
+// RUN: %clang_cc1 -triple x86_64-apple-macos11 \
+// RUN:   -resource-dir %S/Inputs/toolchain_dir/lib/clang/1000 -internal-isystem %S/Inputs/toolchain_dir/lib/clang/1000/include \
+// RUN:   -isysroot %S/Inputs/SDK -internal-externc-isystem %S/Inputs/SDK/usr/include \
+// RUN:   -emit-obj %t/src/main.c -o %t/out/main.o -include %t/src/prefix.h -I %t/src/inc \
+// RUN:   -MT deps -dependency-file %t/regular1.d
+// RUN: %clang_cc1 -triple x86_64-apple-macos11 \
+// RUN:   -resource-dir %S/Inputs/toolchain_dir/lib/clang/1000 -internal-isystem %S/Inputs/toolchain_dir/lib/clang/1000/include \
+// RUN:   -isysroot %S/Inputs/SDK -internal-externc-isystem %S/Inputs/SDK/usr/include \
+// RUN:   -emit-obj %t/src2/main.c -o %t/out2/main.o -include %t/src2/prefix.h -I %t/src2/inc \
+// RUN:   -MT deps -dependency-file %t/regular2.d
+
+// FIXME: The caching compilations include system headers even though they were not requested. Fix and enable `RUN` lines.
```

Comment:
nit: I will leave this out and add it in the other patch that fixes the system header problem.

Also as mentioned in previous review, I prefer directly FileCheck `.d` file if that is not too tricky to write, rather than run `diff` and `CHECK-DEPS-NOT` for a single string.

Reply:
> Also as mentioned in previous review, I prefer directly FileCheck .d file if that is not too tricky to write, rather than run diff and CHECK-DEPS-NOT for a single string.

I intend to do that for the other test, but this test is generally more like "make sure the outputs of the caching compilation are exactly the same as the normal compilation" so the `diff` makes sense here. And the `CHECK-DEPS-NOT` check is temporary.

Reply:
I removed the extra FIXME lines for now.